core/includes/form.inc

```diff
@@ -2876,6 +2876,7 @@ function form_select_options($element, $choices = NULL) {
   $value_is_array = $value_valid && is_array($element['#value']);
   $options = '';
   foreach ($choices as $key => $choice) {
+    $disabled = '';
```

Comment:
Since $disabled is only used within the scope of the `else{}` below, I suggest moving the initialization to within the `else{}`